tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
```

Comment:
Maybe we should just put a style block in tf-event-dashboard?

Renato you own refactoring the dashboards so I'll let you decide what, if anything, should happen here :)